code/datums/mood_events/generic_negative_events.dm

```diff
@@ -17,7 +19,7 @@
 /datum/mood_event/burnt_thumb
 	description = "<span class='warning'>I shouldn't play with lighters...</span>\n"
 	mood_change = -1
-	timeout = 1200
+	timeout = TWOMINUTES
```

Comment:
```suggestion
	timeout = 2 MINUTES
```

all of these have been converted on TG already https://github.com/tgstation/tgstation/blob/master/code/datums/mood_events/generic_negative_events.dm